language/namespaces.xml

```diff
@@ -59,15 +59,15 @@ version="1.1">
     </listitem>
     <listitem>
      <simpara>
-      Die Möglichkeit, um einen Alias (oder einen Kurznamen) für Sehr_Lange_Namen
-      einzuführen, die angelegt wurden, um das erste Problem zu umgehen. Damit kann
+      Die Möglichkeit, einen Alias (oder einen Kurznamen) für Sehr_Lange_Namen
+      einzuführen, die angelegt wurde, um das erste Problem zu umgehen. Damit kann
```

Comment:
Either `die angelegt wurden` or `der angelegt wurde`. I think the former is correct.

Reply:
No, frankly, it refers to "Möglichkeit", which is singular.

Reply:
The `um einen Alias (oder einen Kurznamen) für Sehr_Lange_Namen einzuführen` is a sentence that got "shoved" into the real sentence being: `Die Möglichkeit, die angelegt wurde, um das erste Problem zu umgehen.` Therefore `die angelegt wurde` refers to `Die Möglichkeit`, which is female singular.

Reply:
Ah, right! Maybe better `geschaffen wurde` or `eingeführt wurde` than `angelegt wurde`?

Reply:
I agree, `geschaffen wurde` sounds better. How and where would I change that? In my local fork? Or somehow here?